packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -87,6 +102,97 @@ export default class PciProjectNewPaymentCtrl {
       });
   }
 
+  displayCucCloudMessage(type, suffix) {
+    this.CucCloudMessage[type](
+      this.$translate.instant(
+        `pci_project_new_payment_check_anti_fraud_case_${suffix}`,
+      ),
+      'pci.projects.new.payment',
+    );
+  }
+
+  displayAntiFraudMessage(validatingStep, order) {
+    const {
+      FRAUD_REFUSED,
+      FRAUD_DOCS_REQUESTED,
+      FRAUD_MANUAL_REVIEW,
+    } = ORDER_FOLLOW_UP_STEP_VALIDATING_HISTORY.ENUM;
+
+    (validatingStep?.history || []).forEach(({ label }) => {
+      if (label === FRAUD_REFUSED) {
+        this.displayCucCloudMessage('error', label.toLowerCase());
+      }
+      if ([FRAUD_MANUAL_REVIEW, FRAUD_DOCS_REQUESTED].includes(label)) {
+        this.orderBillingUrl = this.buildOrderBillingUrl(order);
+        this.needToCheckCustomerInformartions = true;
+        this.displayCucCloudMessage('warning', label.toLowerCase());
+      }
+    });
+  }
+
+  static validatingStepIsDone(validatingStep) {
+    return validatingStep.status === ORDER_FOLLOW_UP_STATUS_ENUM.DONE;
+  }
+
+  static isAntiFraudCases(validatingStep) {
+    const {
+      FRAUD_REFUSED,
+      FRAUD_DOCS_REQUESTED,
+      FRAUD_MANUAL_REVIEW,
+    } = ORDER_FOLLOW_UP_STEP_VALIDATING_HISTORY.ENUM;
+
+    return !!validatingStep.history.find(({ label }) =>
+      [FRAUD_REFUSED, FRAUD_MANUAL_REVIEW, FRAUD_DOCS_REQUESTED].includes(
+        label,
+      ),
+    );
+  }
+
+  buildOrderBillingUrl(order) {
```

Comment:
You can use Destructuring assignment here.

see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#unpacking_fields_from_objects_passed_as_a_function_parameter

```suggestion
  buildOrderBillingUrl({ orderId }) {
```

It will allow you to use:
```js
{ orderId }

// instead of

{ orderId: order.orderId  }
```

Reply:
Done